lib/garagemanager.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright reelyActive 2019 - 2019
+ * We believe in an open Internet of Things
+ */
+
+
+const RSSI_MIN_THRESHOLD = -75;
+const RSSI_MAX_THRESHOLD = -30;
+
+/**
+ * GarageManager class
+ * Manages the Garage hlc-dashboard demo
+ */
+class GarageManager {
+
+  /**
+   * GarageManager constructor
+   * @param {Object} options the options as a JSON object
+   * @param {DataBaseManager} database the datbase manager
+   */
+  constructor(options, database){
+    options = options || {};
+
+    this.esClient = options.esClient;
+    this.database = database;
+
+  }
+  /**
+   * gates story id for a specific beacon
+   * @param {string} beaconId specific id of a bluetooth beacon
+   * @param {callback} callback fucntion to return
+   */
+  mapBeaconToStory (beaconId, callback){ //TO BE USED INSTEAD OF retrieveStory WHEN CORMRANT COR ISSUES ARE RESOLVED
+    //TO-DO: map specific stories ids obtained from json-silo to the beaconIds
+    let storyMap = {

```

Comment:
these IDs are the same as in the web/. they can be removed if the stories are to be created with json-silo
we could allow the ids to be created added by the user on the client-side